packages/aws-cdk-lib/aws-ecs/lib/cluster.ts

```diff
@@ -631,6 +622,31 @@ export class Cluster extends Resource implements ICluster {
     }
   }
 
+  private configureAmazonECSOptimizedAMIUserData(autoScalingGroup: autoscaling.AutoScalingGroup, options: AddAutoScalingGroupCapacityOptions = {}) {
+    autoScalingGroup.addUserData(`echo ECS_CLUSTER=${this.clusterName} >> /etc/ecs/ecs.config`);
+    if (!options.canContainersAccessInstanceRole) {
+      // Deny containers access to instance metadata service
+      // Source: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/instance_IAM_role.html
+      autoScalingGroup.addUserData('sudo iptables --insert FORWARD 1 --in-interface docker+ --destination 169.254.169.254/32 --jump DROP');
+
+      switch (options.machineImageType) {
+        case MachineImageType.AMAZON_LINUX_2023: {
+          autoScalingGroup.addUserData('sudo iptables-save');
+          break;
```

Comment:
Can you confirm for `AL2023`, the old command `sudo service iptables save` doesn't work to make sure this is not a breaking change?

Reply:
![image](https://github.com/user-attachments/assets/e6526d74-3076-48f0-8e4a-b4677d946e8e)

Thanks for your review @GavinZZ. This is the old command on latest ECS optimized AL2023 `ami-0a357ea20d7b79c2c`